yt/odin/checks/lib/system_quotas/__init__.py

```diff
@@ -180,6 +180,9 @@ def get_threshold_from_schedule(schedule: ScheduleOptionsDict):
 
 def do_run_check(yt_client, logger, states, options, facade, config):
     cluster = get_cluster_name(yt_client)
+
+    default_threshold = options.get("default_threshold", config.DEFAULT_THRESHOLD)
```

Comment:
nit: default_threshold -> threshold

Reply:
since it's a value that can be override on an account basis, I think default_threshold is a better name